macros/stitch/base/stitch_fb_ad_insights.sql

```diff
@@ -9,7 +9,7 @@
 
 select
 
-    date_start::date as date_day,
+    {{ dbt_utils.date_trunc('day', 'date_start') }} as date_day,
```

Comment:
our dbt doesn't recognise the `::` syntax?

Reply:
This is a type cast operator which is not supported in BigQuery. 

This would work in Postgres (https://www.postgresqltutorial.com/postgresql-cast/) or Redshift (https://docs.aws.amazon.com/redshift/latest/dg/r_CAST_function.html) but not in BigQuery 😞 .